lib/LaTeXML.pm

```diff
@@ -682,23 +680,11 @@ sub flush_log {
   $$self{log} = q{};
   return $log; }
 
-sub sanitize {
-  my ($self, $log) = @_;
-  if ($log =~ m/^Fatal:internal/m) {
```

Comment:
this check almost never (truly never?) was true, so the entire call is pointless.